transform-plugins/docs/JSONParser-transform.md

```diff
@@ -6,13 +6,20 @@ Parses an input JSON event into a record. The input JSON event could be either a
 string fields to values or it could be a complex nested JSON structure. The plugin allows you
 to express JSON paths for extracting fields from complex nested input JSON.
 
-## Parsing Simple JSON
+### Use Case
+JSON Parser reads the values of input column record by record, parses it as json and then extracts the values from json as per output schema.
```

Comment:
We need to explicitly mention that a values of the input column should be a JSON string.